cli/commands/update_test.go

```diff
@@ -0,0 +1,152 @@
+	doDescribe()
+	// assert that request contains our app-id
+	// TODO
+	// assert that printed output is the resolvedOptions field from the JSON
+	expectedOutput := suite.loadFile("testdata/output/describe.txt")
+	assert.JSONEq(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+
+func (suite *UpdateTestSuite) TestDescribeNoOptions() {
+	suite.responseBody = suite.loadFile("testdata/responses/cosmos/1.10/open/describe.json")
+	doDescribe()
+	// assert that user receives an error message
+	expectedOutput := suite.loadFile("testdata/output/no-stored-options.txt")
+	assert.Equal(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+
+func (suite *UpdateTestSuite) TestPrintStatus() {
+	printStatus()
+	expectedOutput := suite.loadFile("testdata/output/status.txt")
+	assert.Equal(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+func (suite *UpdateTestSuite) TestUpdateConfiguration() {
+	suite.responseBody = suite.loadFile("testdata/responses/cosmos/1.10/enterprise/update.json")
+	doUpdate("testdata/input/config.json", "")
+
+	// assert request is what we expect
+	expectedRequest := suite.loadFile("testdata/requests/update-configuration.json")
+	assert.JSONEq(suite.T(), string(expectedRequest), string(suite.requestBody))
+
+	// assert CLI output is what we expect
+	expectedOutput := suite.loadFile("testdata/output/update.txt")
+	assert.Equal(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+
+func (suite *UpdateTestSuite) TestUpdatePackageVersion() {
+	suite.responseBody = suite.loadFile("testdata/responses/cosmos/1.10/enterprise/update.json")
+	doUpdate("", "stub-universe")
+
+	// assert request is what we expect
+	expectedRequest := suite.loadFile("testdata/requests/update-package-version.json")
+	assert.JSONEq(suite.T(), string(expectedRequest), string(suite.requestBody))
+
+	// assert CLI output is what we expect
+	expectedOutput := suite.loadFile("testdata/output/update.txt")
+	assert.Equal(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+
+func (suite *UpdateTestSuite) TestUpdateConfigurationAndPackageVersion() {
+	suite.responseBody = suite.loadFile("testdata/responses/cosmos/1.10/enterprise/update.json")
+	doUpdate("testdata/input/config.json", "stub-universe")
+
+	// assert request is what we expect
+	expectedRequest := suite.loadFile("testdata/requests/update.json")
+	assert.JSONEq(suite.T(), string(expectedRequest), string(suite.requestBody))
+
+	// assert CLI output is what we expect
+	expectedOutput := suite.loadFile("testdata/output/update.txt")
+	assert.Equal(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+
+func (suite *UpdateTestSuite) TestUpdateWithEmptyFile() {
+	suite.responseBody = suite.loadFile("testdata/responses/cosmos/1.10/enterprise/update.json")
+	doUpdate("testdata/input/empty.json", "")
+	expectedOutput := suite.loadFile("testdata/output/empty.txt")
```

Comment:
For these cases where it's just a one-liner error, could they just be inlined?

e.g. here it's `Failed to load specified options file testdata/input/empty.json: unexpected end of JSON input`